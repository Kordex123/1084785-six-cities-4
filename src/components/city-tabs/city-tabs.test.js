import React from "react";
import renderer from "react-test-renderer";
import CityTabs from "./city-tabs";

const mockCityNames = [`Amsterdam`, `Zurich`, `Moscow`, `Kabul`];

it(`Should CityTabs render correctly`, () => {
  const tree1 = renderer
    .create(
        <CityTabs
          cityNames={mockCityNames}
        />
    )
    .toJSON();

  expect(tree1).toMatchSnapshot();
});
