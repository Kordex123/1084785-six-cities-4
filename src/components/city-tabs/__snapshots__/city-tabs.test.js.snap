// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CityTabs render correctly 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Zurich
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Moscow
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Kabul
      </span>
    </a>
  </li>
</ul>
`;
